walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -285,6 +285,8 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 To use the Fuse connections, create an integration in Red Hat Fuse Online.
 
+. Log in to the link:{fuse-url}[Red Hat Fuse Online, window="_blank", id="wt1a_2_3_1_fuse-url"] console.
```

Comment:
@tonyxrmdavidson  I've reused an existing id, is that ok?

Reply:
chatted with @tonyxrmdavidson  - cannot use this id, and naming ids needs to be revised and documented